boilerplate/openshift/golang-osd-operator/csv-generate/catalog-publish.sh

```diff
@@ -41,6 +41,11 @@ BUNDLE_DIR="${SAAS_OPERATOR_DIR}/${operator_name}"
 OPERATOR_NEW_VERSION=$(ls "${BUNDLE_DIR}" | sort -t . -k 3 -g | tail -n 1)
 OPERATOR_PREV_VERSION=$(ls "${BUNDLE_DIR}" | sort -t . -k 3 -g | tail -n 2 | head -n 1)
 
+if [[ "$OPERATOR_NEW_VERSION" == "$OPERATOR_PREV_VERSION" ]]; then
+    echo "New version and previous version are identical. Exiting."
+    exit 1
```

Comment:
```suggestion
    exit 0
```
Depending on the use-case, do we want to `exit 0` here as a success? Or should this be considered a CI failure?

Reply:
This should be considered a failure. This will be triggered if the first part of the pipeline doesn't run correctly.